drivers/base/regmap/Kconfig

```diff
@@ -4,7 +4,7 @@
 # subsystems should select the appropriate symbols.
 
 config REGMAP
-	default y if (REGMAP_I2C || REGMAP_SPI || REGMAP_SPMI || REGMAP_W1 || REGMAP_AC97 || REGMAP_MMIO || REGMAP_IRQ || REGMAP_SOUNDWIRE || REGMAP_SCCB || REGMAP_I3C)
+	default y if (REGMAP_I2C || REGMAP_SPI || REGMAP_SPMI || REGMAP_W1 || REGMAP_AC97 || REGMAP_MMIO || REGMAP_IRQ || REGMAP_SOUNDWIRE || REGMAP_SOUNDWIRE_MBQ || REGMAP_SCCB || REGMAP_I3C)
```

Comment:
does it actually make sense to have `REGMAP_SOUNDWIRE_MBQ` enabled and `REGMAP_SOUNDWIRE` disabled?

Reply:
Good question. I actually don't think it's possible, the MBQs are only used for some volume/gain controls, so we would need regular regmap for other cases.
That said, the code is independent so there's no reason to make the two depend on each other.

Reply:
added comment in commit message